packages/client/hooks/useModalPortal.tsx

```diff
@@ -81,7 +81,7 @@ const ModalContents = styled('div')<{portalStatus: PortalStatus}>(({portalStatus
   display: 'flex',
   flex: '0 1 auto',
   flexDirection: 'column',
-  position: 'relative',
+  position: 'fixed',
```

Comment:
-1 `useModalPortal` is being used by `useModal`, which is used in 36 files. I don't think we should change the styles here, or we'll need to spend a long time making sure that all the other modals work as expected